conf/config.js

```diff
@@ -118,15 +118,50 @@ module.exports = {
      */
 /*
     adminKeys: [
-        //"[cryptpad-user1@my.awesome.website/YZgXQxKR0Rcb6r6CmxHPdAGLVludrAF2lEnkbx1vVOo=]",
+        "[cryptpad-user1@my.awesome.website/YZgXQxKR0Rcb6r6CmxHPdAGLVludrAF2lEnkbx1vVOo=]",
     ],
 */
 
+    /*  CryptPad's administration panel includes a "support" tab
+     *  wherein administrators with a secret key can view messages
+     *  sent from users via the encrypted forms on the /support/ page
+     *
+     *  To enable this functionality:
+     *    run `node ./scripts/generate-admin-keys.js`
+     *    save the public key in your config in the value below
+     *    add the private key via the admin panel
+     *    and back it up in a secure manner
+     *
+     */
+    supportMailboxPublicKey: '',
```

Comment:
I was thinking if we must add this option as default ?

Reply:
I think that this option doesn't do anything unless a key value is set.
I added a bunch of settings that disappeared from the config file in the admin parte such as: `adminEmail`, `removeDonateButton`, `blockDailyCheck`. I don't know why those options got removed from the config.js example file.